.github/ISSUE_TEMPLATE/feature_request_form.yaml

```diff
@@ -0,0 +1,49 @@
+name: Feature Request Form
+description: Feature Request form for Product Team
+title: "Give this Feature Request a title"
+body:
+  - type: dropdown
+    id: choose-platform
+    attributes:
+      label: Platform
+      description: Select from dropdown below.
+      multiple: true
+      options:
+        - PHA-Web
+        - PHA-iOS
+        - PHA-Android
+        - HQ
+        - System
+        - Marketing Website
+    validations:
+      required: true
```

Comment:
I don't think this is needed as the repo the issue originates on will indicate this.

Reply:
removed